sdk/src/tx/utils.ts

```diff
@@ -8,7 +8,7 @@ const COMPUTE_UNITS_DEFAULT = 200_000;
 
 export function wrapInTx(
 	instruction: TransactionInstruction,
-	computeUnits = 500_000 // TODO, requires less code change
+	computeUnits = 600_000 // TODO, requires less code change
```

Comment:
which ix required even more compute?

Reply:
`prepegMarketOrderBaseAssetAmount.ts` but its also updating the amm's which isnt really needed anymore in this branch

so if oracle margin for other positions is deemed good, prior to merge probably need to make the user position markets to read only and remove the `update_amms` for them